app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.find_by_email(session_params[:email])
+    if user&.authenticate(session_params[:password])
+      token = issue_token(user)
+      render json: { user: UserSerializer.new(user), jwt: token }
+    else
+      render json: { error: 'Incorrect username or password.' }
```

Comment:
Let's say `Incorrect email or password` because we're accepting only email and we have no `unique` username.